### ISSUE TITLE
'autohide_bookmarks_and_main_toolbars.css' has a problem since 133

### ISSUE THREAD

**Issue by ilxtx** [MrOtherGuy/firefox-csshacks#482]
Hello, from version 133 to the latest version 134, there is a small problem with this CSS. When clicking the button in the address bar and hovering over the pop-up window that appears, the address bar and bookmark toolbar will be automatically hidden, instead of being displayed continuously. See the picture below for details: 

![autohide_bookmarks_and_main_toolbars css-issus](https://github.com/user-attachments/assets/c579e924-9576-472c-8c90-a06e2a7b24d6)

I like this CSS very much and hope you can fix this problem, thank you very much.

**Comment by MrOtherGuy** [comment]
This is potentially an easy fix, the only complication is that if I do it then the toolbars might also show in some unexpected circumstances which wouldn't be great.

**Comment by ilxtx** [comment]
> This is potentially an easy fix, the only complication is that if I do it then the toolbars might also show in some unexpected circumstances which wouldn't be great.

Thank you for your reply. 

Yes, there is a situation you mentioned. I hope there can be a perfect solution. This CSS was very perfect in versions 132 and earlier, but for some reason, this issue has been present since version 133. 

Thank you again for this very practical project.

**Comment by MrOtherGuy** [comment]
I have now changed a few selectors that should hopefully make more of the panels to keep toolbars visible, hopefully without causing annoying disturbances.

**Comment by ilxtx** [comment]
> I have now changed a few selectors that should hopefully make more of the panels to keep toolbars visible, hopefully without causing annoying disturbances.

Thank you very much. It's very perfect and has passed testing in versions 132 and 134.

**Comment by konsulas** [comment]
I have another issue since version 133. 
I use "autohide_bookmarks_and_main_toolbars.css" with couple modified settings:

#navigator-toolbox{
  --uc-bm-padding: 4px; /* Vertical padding to be applied to bookmarks */
  --uc-bm-height: calc(20px + 2 * var(--uc-bm-padding)); /* Might need to adjust if the toolbar has other buttons */
  --uc-navbar-height: -40px; /* navbar is main toolbar. Use negative value */
  --uc-autohide-toolbar-delay: **300ms**; /* The toolbar is hidden after 0.6s */

and

/* Show when cursor is over the toolbar area */
#navigator-toolbox:hover > .browser-toolbar{
  transition-delay: **1200ms** !important;
  transform: rotateX(0);

Now when mouse pointer is hovering over tab bar, address bar is opening immediately without delay and in very weird way

![1](https://github.com/user-attachments/assets/7b33f928-efde-4481-88af-735451c0f6f8)

 and only after 1200ms delay bookmark toolbar shows up.

![2](https://github.com/user-attachments/assets/78acee89-d712-4f8c-8bb4-6cc4e4c5fe3e)

**Comment by MrOtherGuy** [comment]
Urlbar needs to be transformed separately ,so you'll have to also modify the delay for  urlbar [here on line 74](https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/autohide_bookmarks_and_main_toolbars.css#L74).

**Comment by konsulas** [comment]
> Urlbar needs to be transformed separately ,so you'll have to also modify the delay for urlbar [here on line 74](https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/autohide_bookmarks_and_main_toolbars.css#L74).

Working! Thanks.

**Comment by paulscholes0258** [comment]
@MrOtherGuy 
Hi Bro. how to use autohide_bookmarks_and_main_toolbars.css + autohide_tabstoolbar_v2.css in 1 css.
each of both separate css working good but when I copy combine ( copy text) 2 css in one, it's not working. thanks

**Comment by MrOtherGuy** [comment]
Simply don't. Since you want to autohide all toolbars you should use [autohide_toolbox.css](https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/autohide_toolbox.css) instead.
